### ISSUE TITLE
It would be nice to have the reverse of this code, a compressor

### ISSUE THREAD

**Issue by meszaros-lajos-gyorgy** [tec27/implode-decoder#1]
Hi!

I started using your code for a very crude Arx Fatalis level loader and it works like a charm! Thank you very much for your work!

However, it would be nice, if you could write the compression counterpart of the code, so that I could pack data back. Do you have plans for making that anytime in the future?

Thanks very much!

**Comment by tec27** [comment]
Hey, it's unlikely that I'll get around to that any time soon. It *may* be something I need for other projects, but that hasn't happened as of yet. I think my best recommendation would be to look at [StormLib's implementation](https://github.com/ladislav-zezula/StormLib/blob/master/src/pklib/implode.c)

**Comment by meszaros-lajos-gyorgy** [comment]
Thank you for your response! I've already looked into the C implementation and I'll try writing a node version of it then. If I manage to create a working compressor, then I'll make a pull request for your repo.

**Comment by meszaros-lajos-gyorgy** [comment]
After a few months of grinding through the C code and debugging my implementation based on your code I'm happy to say, that I'm on track with the compression algorhythm. https://github.com/meszaros-lajos-gyorgy/node-pkware

Your repo helped out tremendously and I borrowed your test files for unit testing my own solution. If you are unhappy with it, then I'll immediately delete them from my repo.

I've also added your repo as a source for my work.

So I guess I will close this ticket.
